### ISSUE TITLE
[decomp][bad accuracy] AlbertForQuestionAnswering

### ISSUE THREAD

**Issue by anijain2305** [pytorch/pytorch#103402]
### 🐛 Describe the bug

Repro - `python benchmarks/dynamo/huggingface.py  --backend=aot_eager_decomp_partition --amp --training --device cuda  --accuracy --only=AlbertForQuestionAnswering`

Setup - Get my branch - `https://github.com/pytorch/pytorch/tree/tb-pin` and run the above cmd.

Note that the accuracy fails with `aot_eager_decomp_partition`. It passes with `aot_eager`

My branch
* removes all the decomps except softmax from the decomp table.
* further fires off decomp only for the first softmax.

This limits the scope to just one decomp. But, I am out of ideas. I am unable to debug this further. The softmax decomp looks really simple. Would love to have someone look into this.

### Versions

N/A

cc @ezyang @msaroufim @wconstab @bdhirsh @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @Xia-Weiwen @wenzhe-nrv @jiayisunx @peterbell10 @ipiszy @ngimel @yf225 @chenyang78 @kadeng @muchulee8

**Comment by anijain2305** [comment]
cc @ezyang @bdhirsh @ngimel @desertfire

**Comment by ngimel** [comment]
Is there anything special about softmax inputs for the failing case? Softmax decomposition is used literally everywhere

**Comment by anijain2305** [comment]
Unable to retrieve the inputs. Sadly, minifier could not do anything here.

**Comment by davidberard98** [comment]
@anijain2305 I actually can't repro this locally (although I see that the dashboard is still marking it as failing accuracy). Is there anything else I'd need to do to get this to fail?

Currently I'm running this (on viable/strict):
```
$ with-proxy python benchmarks/dynamo/huggingface.py --accuracy --training --amp --backend inductor --disable-cudagraphs --device cuda --only=AlbertForQuestionAnswering
loading model: 0it [00:07, ?it/s]
cuda train AlbertForQuestionAnswering
pass
```

Note, I'm running this on a devgpu w/ CUDA 12

**Comment by eellison** [comment]
I couldnt repro this, and I commented out the higher tolerance for `AlbertForQuestionAnswering` locally.
